### DISCUSSION TITLE
Multiple payment instance create if user quick double tap on payment method list.

### DISCUSSION THREAD

**Discussion by WishalKalkani** [braintree/braintree-android-drop-in#382] [Q&A]
multiple payment instance create if user quick tap on payment method list.
bellow i attached video. for better understand.
Can you please tell me if android side any listener where we can get callback if user click on payment method list item. 
Thank you.

https://drive.google.com/file/d/14kO1igyuehLtv-MM45BhQ5o6q2TCxLYi/view?usp=drivesdk

**Comment by WishalKalkani** [comment]
@bohford @dpirotte @kcboschert @edebill is there any update ?

**Comment by sshropshire** [comment]
Hi @WishalKalkani thanks for using the Braintree SDK for Android. Would you mind creating a new [GitHub issue](https://github.com/braintree/braintree-android-drop-in/issues/new) for this with steps to reproduce the error? It'll help us to properly track progress on a resolution.

**Comment by WishalKalkani** [comment]
@sshropshire i have created. please check

**Comment by scannillo** [comment]
Closing in favor of https://github.com/braintree/braintree-android-drop-in/issues/388
